gnosis/safe-react
# [Gateway SDK] Post transactions using client-gateway

## What it solves
Resolves #2596.

## How this PR fixes it
Imports the Gateway SDK and calls its `postTransaction` method.

It won't work until we migrate the SDK to the "unified" API. The signature should be nullable.
Blocked by https://github.com/gnosis/safe-react-gateway-sdk/issues/1.

This PR also fixes an uncaught exception when the request fails. It's now caught and displayed in a snackbar.

## How to test it
* Create a send funds tx
* Confirm or reject it
